src/main/java/com/webank/wecross/stub/RequestFactory.java

```diff
@@ -0,0 +1,12 @@
+public class RequestFactory {
```

Comment:
It's not a valid java file that no dependent file found. Please import "Request" class.